File: android/pad-java/app/src/main/java/eu/id3/face/samples/padjava/Parameters.java
package eu.id3.face.samples.padjava;

import android.hardware.camera2.CameraCharacteristics;

public class Parameters {
    public final static int detectorThreadCount = 4;
    public final static int detectorConfidenceThreshold = 70;
    public final static int blurScoreMaxThreshold = 20;
    public final static int colorScoreThreshold = 90;
    public final static int colorScoreConfidenceThreshold = 70;
    public final static int maxProcessingImageSize = 512;
    public final static int cameraType = CameraCharacteristics.LENS_FACING_FRONT;
}

File: android/pad/app/src/main/java/eu/id3/face/samples/pad/Parameters.kt
package eu.id3.face.samples.pad

import android.hardware.camera2.CameraCharacteristics

/**
 * Contains the parameters needed in the app. Values of these parameters can be changed and should not be seen as "the perfect" values.
 */
internal object Parameters {
    const val detectorThreadCount = 4
    const val detectorConfidenceThreshold = 70
    const val blurScoreMaxThreshold = 20
    const val colorScoreThreshold = 90
    const val colorScoreConfidenceThreshold = 70
    const val maxProcessingImageSize = 512
    const val cameraType = CameraCharacteristics.LENS_FACING_FRONT
}
